README.md

```diff
@@ -0,0 +1,34 @@
+# Minetest Game
+
+The default game bundled in the Minetest engine.  
+For information about the game, click [here](https://forum.minetest.net/viewtopic.php?f=15&t=9724#p148074).
```

Comment:
That particular post is not official, but the entire topic is. Please drop the id-tag.
Even though the link location is shown on hover, it would be more friendly to write something like this:
`For further information, check this [forum topic](link).`

Reply:
I agree with the above.

Reply:
Please verify if it is correct now.